// node-telegram-keyboard-wrapper/lib/InlineKeyboardButton.js
"use strict";
Object.defineProperty(exports, "__esModule", { value: true });
class InlineKeyboardButton {
    constructor(text, exclusiveKey, exclusiveValue) {
        this.text = text;
        this.exclusiveKey = exclusiveKey;
        this.exclusiveValue = exclusiveValue;
        if (!exclusiveKey || !exclusiveValue) {
            throw new Error("Missing exclusiveKey or exclusiveValue. Creating an InlineKeyboardButton requires one of the mandatory values among optional values. For more refer to: https://core.telegram.org/bots/api#inlinekeyboardbutton");
        }
        Object.freeze(this);
    }
    clone() {
        const exclusiveValue = typeof this.exclusiveValue === "object"
            ? Object.assign({}, this.exclusiveValue)
            : this.exclusiveValue;
        return new InlineKeyboardButton(this.text, this.exclusiveKey, exclusiveValue);
    }
    getJSON() {
        return {
            text: this.text,
            [this.exclusiveKey]: this.exclusiveValue,
        };
    }
}
exports.default = InlineKeyboardButton;
